packages/common/src/store/buy-usdc/sagas.ts

```diff
@@ -393,7 +401,8 @@ function* recoverPurchaseIfNecessary() {
   const audiusBackendInstance = yield* getContext('audiusBackendInstance')
 
   try {
-    yield* call(waitForValue, getFeePayer)
+    const feePayerString: string = yield* call(waitForValue, getFeePayer)
```

Comment:
type needed because the helper is not typed i assume?

Reply:
Correct. Even has a comment about how it would be nice for the helper to be typed :-D